tests/test_files/csv.yaml

```diff
@@ -5,6 +5,16 @@ kind: ClusterServiceVersion
 metadata:
   name: marketplace-operator.v0.0.1
   namespace: placeholder
+  annotations:
+    categories: openshift required
+    certified: "true"
```

Comment:
nit: Noticed for the other test files, the strings in the `certified` field used single quotes rather than double quotes. These should follow a single convention.

Reply:
Addressed in latest commit.